### DISCUSSION TITLE
"sm:block" class is not overwriting "hidden" class after passing the "sm:" breakpoint

### DISCUSSION THREAD

**Discussion by WaterlessPiano2** [tailwindlabs/tailwindcss#2933] [Help] [answered]
I am following the navbar tutorial on youtube and I am stuck on [this part](https://youtu.be/qrTsS3z8BAw?t=43), where the  `sm:block` class should override the `hidden` class when the screen width hits the `sm:` breakpoint. [This almost exactly the same example](https://play.tailwindcss.com/zpW7kLkvp6) works as expected, where the items show when the page's width is increased.

However, when I try to implement this on my project which is created using  `npx create-next-app --example blog-starter-typescript`. [This is where I got it from](https://github.com/vercel/next.js/tree/canary/examples/blog-starter-typescript), the items on the navbar don't show when the page width is increased.

[Here is the exact spot in my repo ](https://github.com/Agile-Import/marketing/pull/1/commits/1cc1bcf4c4c6c9193c92efc0cd57d2db83ab8cf9#r532026926)where this doesn't work. 

Can anyone see what I am doing wrong?

**Comment by WaterlessPiano2** [comment, marked as answer]
"Dark Theme for Chrome" extension has been injecting styling that had `display : hidden !important` . Removing that extension has resolved my issie

**Comment by JanitaM** [comment]
OMG this was driving me crazy. I have/had the same problem - `hidden` was overriding `md:block`. I changed my Windows theme to light and it works. The weird thing is I've had the dark theme turned on forever, and the same code worked a couple of weeks ago.
